engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -94,15 +94,14 @@ <h4 translate>Subscriptions</h4>
           {{ host.subscription_status_label }}
         </dd>
 
-        <dt translate>Details</dt>
+        <dt translate ng-if="!simpleContentAccessEnabled">Details</dt>
         <dd>
```

Comment:
Does this <dd> also need the ng-if check?

Reply:
ack. wonder if it was making it into the HTML...

Reply:
My guess is that since it's looping through `reason in host.subscription_facet_attributes.compliance_reasons`, the `dd` and `ul` were there but nothing was visible, since my host didn't happen to have any compliance reasons.